code/modules/clothing/masks/gasmask.dm

```diff
@@ -94,9 +94,6 @@ GLOBAL_LIST_INIT(clown_mask_options, list(
 	var/valid_wearer = ismob(loc)
 	var/mob/wearer = loc
 	if(istype(tool, /obj/item/cigarette))
-		if(flags_cover & MASKCOVERSMOUTH)
-			balloon_alert(user, "mask's mouth is covered!")
-			return ..()
```

Comment:
Maybe supposed to be checking if the mob is wearing a helmet that covers the mouth? Or supposed to check stuff like miner gas masks which can be flipped up? Idk

Reply:
Yeah the mining mask was part of my tests while fiddling around with this, this allows it to use cigarettes while it isn't folded for storage/mouth is uncovered and you can eat. Which is how it normally should be really if you consider the fact that you stuff the cigarette in the filter slot and not in your mouth. 

As for a helmet that covers that mouth, I figured it'd be okay because you can still smoke with a helmet that covers covers your mouth, so long as you just do it directly via the mask slot and not through the filter slot on the mask on the mask slot.

But if this reasoning is not acceptable, then please do tell me how else I would fix this without removing the flag for coving the mouth of the gas mask, which I feel like would reach balance change levels